src/amp-user-notification_with_local_storage.html

```diff
@@ -33,12 +34,12 @@
   <!-- #### Using Local Storage -->
   <!--
     When using the local storage solution, the parameters data-show-if-href and
-    data-dismiss-if-href are optional. When the user agrees to the cookies policy,
-    a new item is added to the local storage.
+    data-dismiss-if-href are optional. 
```

Comment:
Remove this sentence. If users read this article first,  they don't know what these parameters are.